src/Tizen.NUI.Components/Controls/Slider.cs

```diff
@@ -926,8 +920,7 @@ public string ValueIndicatorUrl
         /// Then, the thumb can only go to 0, 20, 40, 60, 80, and 100.
         /// The default is 0.
         /// </summary>
-        /// This will be public opened later after ACR done. Before ACR, need to be hidden as inhouse API.
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
```

Comment:
The detailed description would better fit a \<details\> section. I mean I would suggest splitting the summary: leave first 2 sentences in \<summary\>, put the rest in \<details\>.

Reply:
I update this in another PR : https://github.com/Samsung/TizenFX/pull/2990